bigtable/api/Snippets/Reads/PrintRow.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+using Google.Cloud.Bigtable.V2;
+using System;
+
+namespace Reads
```

Comment:
The same function for printing can be used for the read and filtering samples

Reply:
@billyjacobson  Thanks for the feedback. I already discuss with @amanda-tarafa internally We are fully re-writing the samples and currently I am working on it.
I am closing this PR and create another soon.